<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1938</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255))</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>820</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>831</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 161, 30, 255), stop:0.5 rgba(110, 250, 60, 255), stop:1 rgba(96, 199, 193, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color :qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255))</string>
    </property>
    <property name="text">
     <string> P h o t o S h o p </string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="grayscale">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>730</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>grayscale_cv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayscale2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>grayscale_qt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blur_g">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>blur_gaussian</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sharpen_cv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>sharpen_cv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalization">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>equalization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undo">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redo">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="background">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>background</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_confirm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="word">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>170</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))</string>
    </property>
    <property name="text">
     <string>basic effects</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrast_color">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>contrast_color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resize_d">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>dpi_deprove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resize_i">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>dpi_improve</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_saturation">
    <property name="geometry">
     <rect>
      <x>1319</x>
      <y>260</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saturation">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>160</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>satuation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightness">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>160</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>brightness</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_brightness">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>260</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="warm">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>340</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>warm</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_warm">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>440</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cold">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>340</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>cold</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_cold">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>440</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="contrast">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>520</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>contrast</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_contrast">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>620</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="map_turn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>640</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>map_turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mirror">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>730</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>mirror</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>820</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>810</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Harlow Solid Italic</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_exit">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>780</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1938</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionexit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
